superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -29,16 +29,18 @@ import {
   SortIndicator,
   Table,
 } from 'react-virtualized';
-import { getTextDimension } from '@superset-ui/dimension';
+import { createSVGNode, getTextDimension } from '@superset-ui/dimension';
 import { t } from '@superset-ui/translation';
 
 import Button from '../Button';
 import CopyToClipboard from '../CopyToClipboard';
 import ModalTrigger from '../ModalTrigger';
 import TooltipWrapper from '../TooltipWrapper';
 
+const svg = createSVGNode({ container: document.body });
```

Comment:
does this ever get removed from the DOM? Maybe it would be better to create the node in the ` getWidthsForColumns` function and remove it from the DOM after everything is measured

Reply:
Ah, good point. I'll do that.